cfme/tests/configure/test_tag.py

```diff
@@ -284,7 +319,13 @@ def test_assign_and_unassign_tag(self, appliance, tags_mod, provider, services_m
             pytest.skip("No available entity in {} to assign tag".format(collection_name))
         entity = collection[-1]
         tag = tags_mod[0]
-        entity.tags.action.assign(tag)
+        try:
+            entity.tags.action.assign(tag)
+        except AttributeError:
+            msg = ('Missing tag attribute in parametrized REST collection {} for entity: {}'
```

Comment:
Keying please

Reply:
There's only two fields, its clear. I don't think we need to key in every format() call. its useful when there are many fields to fill.